python/seldon_core/wrapper.py

```diff
@@ -1,27 +1,29 @@
 import grpc
+import os
+import logging
+import seldon_core.seldon_methods
+
 from concurrent import futures
-from flask import jsonify, Flask, send_from_directory, request, Response
+from flask import Flask, send_from_directory, request, Response
 from flask_cors import CORS
-import logging
 from seldon_core.utils import (
     seldon_message_to_json,
     json_to_seldon_model_metadata,
     json_to_feedback,
+    getenv_as_bool,
 )
-from seldon_core.flask_utils import get_request
-import seldon_core.seldon_methods
+from seldon_core.flask_utils import get_request, jsonify
 from seldon_core.flask_utils import (
     SeldonMicroserviceException,
     ANNOTATION_GRPC_MAX_MSG_SIZE,
 )
 from seldon_core.proto import prediction_pb2_grpc
-from seldon_core.proto import prediction_pb2
-import os
 
 logger = logging.getLogger(__name__)
 
 PRED_UNIT_ID = os.environ.get("PREDICTIVE_UNIT_ID", "0")
 METRICS_ENDPOINT = os.environ.get("PREDICTIVE_UNIT_METRICS_ENDPOINT", "/metrics")
+PAYLOAD_PASSTHROUGH = getenv_as_bool("PAYLOAD_PASSTHROUGH", default=False)
```

Comment:
Would be good to document this in the python section and in the upgrading page

Reply:
By default that flag will be disabled, so the behaviour will be the same as it is currently. We can add a note on the docs though! I can see how it could be useful in some use cases outside of this PR.